src/main/java/org/motechproject/newebodac/repository/FieldConfigRepository.java

```diff
@@ -4,10 +4,18 @@
 import java.util.UUID;
 import org.motechproject.newebodac.domain.FieldConfig;
 import org.motechproject.newebodac.domain.enums.EntityType;
+import org.motechproject.newebodac.dto.BaseDto;
+import org.motechproject.newebodac.exception.NewEbodacException;
+import org.motechproject.newebodac.repository.custom.FieldConfigRepositoryCustom;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.repository.query.Param;
 
-public interface FieldConfigRepository extends JpaRepository<FieldConfig, UUID> {
+public interface FieldConfigRepository extends JpaRepository<FieldConfig, UUID>, FieldConfigRepositoryCustom {
 
   List<FieldConfig> findByEntity(@Param("entityType") EntityType entityType);
+
+  BaseDto search(EntityType entity, BaseDto value) throws NewEbodacException;
```

Comment:
it's already in FieldConfigRepositoryCustom, so there is no need to have the declaration here too